lib/util.js

```diff
@@ -301,10 +301,10 @@ function ensureDebugIsInitialized() {
 
 
 function inspectPromise(p) {
-  ensureDebugIsInitialized();
   // Only create a mirror if the object is a Promise.
```

Comment:
I guess to me it's a little confusing since the comment is talking about creating a mirror which isn't done on the next line and the function name does not indicate that a mirror is being created. \*shrug\*